iptables/table.go

```diff
@@ -29,7 +29,8 @@ import (
 )
 
 const (
-	MaxChainNameLength = 28
+	MaxChainNameLength       = 28
+	defaultPostWriteInterval = 50 * time.Millisecond
```

Comment:
This is really a minimum, isn't it, so `minPostWriteInterval` perhaps?

IIUC it never gets used as a default, because the default IptablesPostWriteCheckInterval is 1s, which is larger than this.